package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.8.1",
```

Comment:
- [ ] `redux-thunk` was supposed to have been added to the project as a package and used as a middleware <= Would you add it?

Reply:
Hi, @descholar-ceo. As seen in my package.json file. I have installed the new reduxJS toolkit instead. This lets me have access to redux-thunk and apply middleware without installing those packages independently

Reply:
It should be used as a middleware then in the store

Reply:
@descholar-ceo 
I have access to this by default using the reduxjs toolkit. it's redundant to install them again
[see here](https://redux.js.org/usage/configuring-your-store#simplifying-setup-with-redux-toolkit)
![image](https://user-images.githubusercontent.com/55185309/167954437-aca7b68d-b6c3-4de1-b964-8aae9c5ab6b3.png)

Reply:
I see your point, your project should be ready then!